src/content/docs/guides/usage/data/shape-data.mdx

```diff
@@ -225,3 +230,54 @@ sort -x
 ```
 
 Prepending the field with `-` reverses the sort order.
+
+## Use functions to change single events
+
+Functions work on single values, including the top-level event that you can
+reference with the `this` keyword.
+
+### Combine records with `merge`
```

Comment:
You can probably copy this paragraph and almost replace records with lists and it'll work. Note that for `list` we have three functions: `concatenate`, `append`, and `prepend`, which are equal to `[...lhs, ...rhs]`, `[...lhs, rhs]`, and `[lhs, ...rhs]`, respectively.